[2023-10-06 16:28:00] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.checkAnswer(TelegramBot.java:157)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:126)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:28:19] - Replied to User: Алекс
[2023-10-06 16:28:24] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.getQuestion(TelegramBot.java:143)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:103)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:28:41] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.checkAnswer(TelegramBot.java:157)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:126)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:30:37] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.getQuestion(TelegramBot.java:143)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:130)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:30:58] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.checkAnswer(TelegramBot.java:157)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:126)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:31:04] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.checkAnswer(TelegramBot.java:157)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:126)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:32:38] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.checkAnswer(TelegramBot.java:157)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:126)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:36:25] - Replied to User: Алекс
[2023-10-06 16:37:19] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.getQuestion(TelegramBot.java:143)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:130)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:37:39] - Index 4 out of bounds for length 4
java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at com.tele.qwest.service.TelegramBot.checkAnswer(TelegramBot.java:157)
	at com.tele.qwest.service.TelegramBot.checking(TelegramBot.java:126)
	at com.tele.qwest.service.TelegramBot.onUpdateReceived(TelegramBot.java:111)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.telegram.telegrambots.meta.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
	at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:317)
[2023-10-06 16:39:36] - Closing JPA EntityManagerFactory for persistence unit 'default'
[2023-10-06 16:39:36] - HikariPool-1 - Shutdown initiated...
[2023-10-06 16:39:36] - HikariPool-1 - Shutdown completed.
[2023-10-06 16:39:44] - Starting App using Java 17.0.8 with PID 14048 (F:\Java\qwest\target\classes started by tjapka in F:\Java\qwest)
[2023-10-06 16:39:44] - No active profile set, falling back to 1 default profile: "default"
[2023-10-06 16:39:44] - Bootstrapping Spring Data JPA repositories in DEFAULT mode.
[2023-10-06 16:39:45] - Finished Spring Data repository scanning in 35 ms. Found 1 JPA repository interfaces.
[2023-10-06 16:39:45] - Tomcat initialized with port(s): 8080 (http)
[2023-10-06 16:39:45] - Initializing ProtocolHandler ["http-nio-8080"]
[2023-10-06 16:39:45] - Starting service [Tomcat]
[2023-10-06 16:39:45] - Starting Servlet engine: [Apache Tomcat/10.1.12]
[2023-10-06 16:39:45] - Initializing Spring embedded WebApplicationContext
[2023-10-06 16:39:45] - Root WebApplicationContext: initialization completed in 968 ms
[2023-10-06 16:39:45] - HHH000204: Processing PersistenceUnitInfo [name: default]
[2023-10-06 16:39:45] - HHH000412: Hibernate ORM core version 6.2.7.Final
[2023-10-06 16:39:45] - HHH000406: Using bytecode reflection optimizer
[2023-10-06 16:39:45] - HHH000021: Bytecode provider name : bytebuddy
[2023-10-06 16:39:45] - No LoadTimeWeaver setup: ignoring JPA class transformer
[2023-10-06 16:39:45] - HikariPool-1 - Starting...
[2023-10-06 16:39:46] - HikariPool-1 - Added connection com.mysql.cj.jdbc.ConnectionImpl@749d7c01
[2023-10-06 16:39:46] - HikariPool-1 - Start completed.
[2023-10-06 16:39:46] - HHH000021: Bytecode provider name : bytebuddy
[2023-10-06 16:39:46] - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
[2023-10-06 16:39:46] - Initialized JPA EntityManagerFactory for persistence unit 'default'
[2023-10-06 16:39:47] - error setting bots command List: Error executing org.telegram.telegrambots.meta.api.methods.commands.SetMyCommands query: [400] Bad Request: BOT_COMMAND_INVALID
[2023-10-06 16:39:47] - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
[2023-10-06 16:39:47] - Starting ProtocolHandler ["http-nio-8080"]
[2023-10-06 16:39:47] - Tomcat started on port(s): 8080 (http) with context path ''
[2023-10-06 16:39:47] - Started App in 3.646 seconds (process running for 4.329)
[2023-10-06 16:41:29] - Closing JPA EntityManagerFactory for persistence unit 'default'
[2023-10-06 16:41:29] - HikariPool-1 - Shutdown initiated...
[2023-10-06 16:41:29] - HikariPool-1 - Shutdown completed.
[2023-10-06 16:41:33] - Starting App using Java 17.0.8 with PID 4532 (F:\Java\qwest\target\classes started by tjapka in F:\Java\qwest)
[2023-10-06 16:41:33] - No active profile set, falling back to 1 default profile: "default"
[2023-10-06 16:41:33] - Bootstrapping Spring Data JPA repositories in DEFAULT mode.
[2023-10-06 16:41:33] - Finished Spring Data repository scanning in 34 ms. Found 1 JPA repository interfaces.
[2023-10-06 16:41:34] - Tomcat initialized with port(s): 8080 (http)
[2023-10-06 16:41:34] - Initializing ProtocolHandler ["http-nio-8080"]
[2023-10-06 16:41:34] - Starting service [Tomcat]
[2023-10-06 16:41:34] - Starting Servlet engine: [Apache Tomcat/10.1.12]
[2023-10-06 16:41:34] - Initializing Spring embedded WebApplicationContext
[2023-10-06 16:41:34] - Root WebApplicationContext: initialization completed in 950 ms
[2023-10-06 16:41:34] - HHH000204: Processing PersistenceUnitInfo [name: default]
[2023-10-06 16:41:34] - HHH000412: Hibernate ORM core version 6.2.7.Final
[2023-10-06 16:41:34] - HHH000406: Using bytecode reflection optimizer
[2023-10-06 16:41:34] - HHH000021: Bytecode provider name : bytebuddy
[2023-10-06 16:41:34] - No LoadTimeWeaver setup: ignoring JPA class transformer
[2023-10-06 16:41:34] - HikariPool-1 - Starting...
[2023-10-06 16:41:35] - HikariPool-1 - Added connection com.mysql.cj.jdbc.ConnectionImpl@69a73867
[2023-10-06 16:41:35] - HikariPool-1 - Start completed.
[2023-10-06 16:41:35] - HHH000021: Bytecode provider name : bytebuddy
[2023-10-06 16:41:35] - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
[2023-10-06 16:41:35] - Initialized JPA EntityManagerFactory for persistence unit 'default'
[2023-10-06 16:41:36] - error setting bots command List: Error executing org.telegram.telegrambots.meta.api.methods.commands.SetMyCommands query: [400] Bad Request: BOT_COMMAND_INVALID
[2023-10-06 16:41:36] - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
[2023-10-06 16:41:36] - Starting ProtocolHandler ["http-nio-8080"]
[2023-10-06 16:41:36] - Tomcat started on port(s): 8080 (http) with context path ''
[2023-10-06 16:41:36] - Started App in 3.689 seconds (process running for 4.387)
[2023-10-06 16:41:42] - Replied to User: Алекс
